docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -2011,35 +2011,35 @@ Here's how to migrate:
  MuiSlider: {
    styleOverrides: {
      root: {
--      [`&.${sliderClasses.thumbSizeSmall}`]: {
```

Comment:
`thumbSizeSmall` class is applied to `thumb`, not `root`. hence, a space has been added here.

docs for same -> https://mui.com/material-ui/api/slider/#slider-classes-MuiSlider-thumbSizeSmall